frontend/src/components/common/OtherCommunitie.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
```

Comment:
It might be better to name `OtherCommunity.tsx` I think

Reply:
Yeah, i agree.